src/styles/blocks/block.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  width: $week-size;
+  margin: 1px;
+  padding: 10px;
+
+  display: flex;
+  flex-wrap: wrap;
+
+  @each $day, $size in $days {
+    .calendar--start-day--#{$day}
+    &__block:first-child {
+      margin: $size;
+    }
```

Comment:
Your calendar shows 31 days, despite having the modifier 29 days active on it. Meaning this loop isn't working. Where is the &__block coming from? And it's a margin-left we are trying to change.